assets/app/view/game/spreadsheet.rb

```diff
@@ -253,26 +277,31 @@ def render_corporation(corporation, operating_order, current_round, index)
             if @game.round.active_step&.did_sell?(corporation, p)
               sold_props[:style][:backgroundColor] = '#9e0000'
               sold_props[:style][:color] = 'white'
+            elsif num_shares_of(p, corporation).zero?
+              sold_props[:style][:opacity] = '0.5'
             end
-            share_holding = corporation.president?(p) ? '*' : ''
-            share_holding += num_shares_of(p, corporation).to_s unless corporation.minor?
-            h('td.padded_number', sold_props, share_holding)
+
+            sold_props[:style][:fontWeight] = 'bold' if corporation.president?(p)
+            share_holding = num_shares_of(p, corporation).to_s unless corporation.minor?
+            share_holding = '*' if corporation.minor? && corporation.president?(p)
+            h('td.padded_number', sold_props, share_holding || '')
           end,
           h('td.padded_number', { style: { borderLeft: border_style } }, num_shares_of(corporation, corporation).to_s),
           h('td.padded_number', { style: { borderRight: border_style } },
             "#{corporation.receivership? ? '*' : ''}#{num_shares_of(@game.share_pool, corporation)}"),
-          h('td.padded_number', corporation.par_price ? @game.format_currency(corporation.par_price.price) : ''),
+          (h('td.padded_number',
+             corporation.par_price ? @game.format_currency(corporation.par_price.price) : '') unless @hide_ipo),
```

Comment:
If @hide_ipo, then it will be nil. Does it show something good?
Alternative would be 
 (h('td.padded_number',
             corporation.par_price && !@hide_ipo ? @game.format_currency(corporation.par_price.price) : '')),

Reply:
I put a `.compact` in the end of the Array to remove the `nil`s usually I'd be appending things to a stores Array conditionally, but since this was in the middle of a very big block I thought that adding a lot of `<<` or concatenating two big Arrays would be more clumsy.

I don't like that alternative very much, because you end up with an empty column in the table.

Reply:
don't use this syntax

Reply:
Which one?

Reply:
Never mind. Got it from the other comment